ethereum/core/src/main/java/org/hyperledger/besu/ethereum/privacy/PrivateTransactionProcessor.java

```diff
@@ -228,14 +222,4 @@ private AbstractMessageProcessor getMessageProcessor(final MessageFrame.Type typ
         throw new IllegalStateException("Request for unsupported message processor type " + type);
     }
   }
-
-  @SuppressWarnings("unused")
-  private long refunded(
```

Comment:
This section had some signed/unsigned errors.  And it is dead code.  Rather than fix it I'm removing it (along with DI tendrils)